src/previewFrame/elementHighlighterScript.js

```diff
@@ -0,0 +1,36 @@
+const elementHighlighterScript = `(${function() {
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    const selector = selectorFromMessage;
+    updateCovers(selector);
+  });
+
+  function updateCovers(selector) {
+    const highlighterElements =
+    window.document.querySelectorAll('.popcode_HIGHLIGHTER');
+    if (highlighterElements.length !== 0) {
+      highlighterElements.forEach((highlighterElement) => {
+        highlighterElement.remove();
+      });
+    }
+
+    if (selector !== '') {
+      const elements =
+      document.querySelectorAll(selector);
```

Comment:
Indent (also this probably doesn’t need to be on its own line?)

Reply:
yup!